### PR TITLE
build passes travis CI

### PR DESCRIPTION
**Pull request by ValHayot**

### CODE REVIEW

File: sim/tests/execute_pytest.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# !/bin/bash
```
Comment (glatard) [inline sim/tests/execute_pytest.sh:1]: couldn't the content of this script be part of the Travis config?

File: sim/tests/test_all.py [hunk 2]
```diff
@@ -53,8 +61,9 @@ def test_spark_bids_skip_participant(self):
       participant_file = "skip.txt"
       with open(participant_file,"w") as f:
          f.write("01")
-      self.run_spark_bids(options=["--skip-participants","skip.txt"],correctBrainSize="865472")
-      
+      self.run_spark_bids(options=["--skip-participants", "skip.txt"],correctBrainSize="865472")
+   
+   @pytest.mark.skipif(not module_exists("hdfs"), reason="HDFS not installed")
```
Comment (glatard) [inline sim/tests/test_all.py]: nice workaround!

File: sim/tests/execute_pytest.sh (hunk 1 above)
Comment (ValHayot) [reply sim/tests/execute_pytest.sh:1]: execute_pytest.sh is being called within the docker container called by travis, not outside of it. I thought about what was the best way to go about it, and this is the best I can think of...for now.  I can place two `docker run...`'s in script of the Travis config and remove this file, but then i guess we'll loose pytest coverage as i don't think i can pass arguments to a docker environment. 

The path should be in the dockerfile though, so i'll fix that!

File: sim/tests/test_all.py (hunk 2 above)
Comment (ValHayot) [reply sim/tests/test_all.py]: yeah, except there's an even better way of doing it. Will fix this now

**Pull request merged**